wp-content/themes/twentyseventeen-wp20/style.css

```
@@ -26,6 +26,7 @@ Template:    twentyseventeen
 	--wp20--color--text-meta: var(--wp20--color--grey-6);
 	--wp20--color--text-link: var(--wp20--color--blueberry);
 	--wp20--color--border: var(--wp20--color--grey-8);
+	--wp20--color--active-menu: #F7F7F7;
```

This semantic color should reference the base color
```suggestion
	--wp20--color--active-menu: var(--wp20--color--grey-9);
```